src/gl/pointFeature.js

```diff
@@ -279,34 +280,34 @@ geo.gl.pointFeature = function (arg) {
     }
 
     sourcePositions.pushBack(buffers.get("pos"));
-    geom.addSource(sourcePositions);
+    geom.addSource(sourcePositions, "pos");
```

Comment:
@aashish24 I've pushed a change to vgl and to here that allows passing additional arguments to all the sources.  I'm happy to change it if you'd rather have something different.